package.json

```diff
@@ -232,6 +232,9 @@
         "tsc-watch": "^4.2.9",
         "xhr-mock": "^2.5.1"
     },
+    "resolutions": {
+        "**/webpack": "^5.0.0"
```

Comment:
I don't love this, but this one line lets us [get rid of >1000 lines of dependencies in one sweep](https://github.com/owid/owid-grapher/pull/1060/commits/c0c84fe147afee7f526777b34148e1d9c1b94a06).
We can remove it once we use Storybook, which will no longer depend on Webpack 4 (https://github.com/storybookjs/storybook/issues/13491)